src/main/java/ProjectOcean/Controller/CourseController.java

```diff
@@ -17,23 +18,29 @@
 /**
  * Represents the visual component of a course
  */
-public class CourseListIconController extends VBox implements Movable {
+
+public class CourseController extends VBox implements Movable {
+
 
     @FXML private Text courseCodeText;
     @FXML private Text courseNameText;
     @FXML private Text studyPointsText;
     @FXML private VBox typeIndicator;
 
+    private Node owner;
```

Comment:
Not necessary as a global variable. This could be removed and on row 128 instead create a Node owner.